tools/docker/builder/openwrt/scripts/build.sh

```diff
@@ -16,3 +16,4 @@ PRPLMESH_VERSION=${PRPLMESH_VERSION}
 EOT
 find bin -name 'prplmesh*.ipk' -exec cp -v {} "artifacts/prplmesh.ipk" \;
 find bin/targets/"$TARGET_SYSTEM"/"$SUBTARGET"/ -type f -maxdepth 1 -exec cp -v {} "artifacts/" \;
+cp .config artifacts/
```

Comment:
Does gitlab-ci render "hidden" files nicely? If not, we may have to rename it, e.g. to `openwrt.config`.

Reply:
I don't know how Gitlab renders them, but I applied your suggestion anyway.